tests/php/ORM/DataListEagerLoadingTest.php

```diff
@@ -696,7 +697,7 @@ private function iterateEagerLoadData(DataList $dataList, int $chunks = 0): arra
                 }
             }
             $s = ob_get_clean();
-            $s = preg_replace('/.*__START_ITERATE__/s', '', $s);
+            $s = preg_replace('/.*__START_ITERATE__/s', '', $s) ?? '';
```

Comment:
```suggestion
            $s = preg_replace('/.*__START_ITERATE__/s', '', $s);
```
If an error occurred I'd prefer a hard error here than simply an empty string

Reply:
This was helpful in debugging - but we can always put this back if we need to debug stuff in the future so I'm fine with removing it.
Done.